ui-tests/cypress/integration/administration/email-configuration-Sender.spec.js

```diff
@@ -0,0 +1,306 @@
+                  .should('be.visible')
+                  .and('have.text', 'Add');
+
+      });
+
+      it([Tag.SMOKE, 'ENG-3299'], 'New Sender is displayed', () => {
+
+        currentPage.getContent().getAddButton().click().wait(1000);
+        cy.validateAppBuilderUrlPathname('/email-config/senders/add');
+        currentPage.getContent().getSenderForm()
+                   .should('be.visible')
+                   .and('contain', 'Code ')
+                   .and('contain', 'Email ');
+
+        currentPage.getContent()
+                   .getCodeInput()
+                   .should('be.visible')
+                   
+        currentPage.getContent()
+                   .getEmailInput()
+                   .should('be.visible');
+
+      });
+      it([Tag.SMOKE, 'ENG-3299'], 'Action Buttons are diplayed', () => {
+         
+
+        currentPage.getContent().getActionButton().click().wait(500);
+
+        currentPage.getContent().getContextMenu().should('be.visible');
+        currentPage.getContent().getContextMenu()
+                    .contains('Edit')
+                    .should('be.visible')
+                    .should('have.text','Edit');
+        currentPage.getContent().getContextMenu()
+                    .contains('Delete')
+                    .should('be.visible')
+                    .should('have.text','Delete');
+
+
+      });
+      it([Tag.SMOKE, 'ENG-3299'], 'Edit Sender is displayed', () => {
+
+
+        currentPage.getContent().getActionButton().click();
+        currentPage.getContent().getEditButton().click().wait(1000);
+        cy.validateAppBuilderUrlPathname('/email-config/senders/edit/CODE1');
+        currentPage.getContent().getSenderForm()
+                   .should('be.visible')
+                   .and('contain', 'Code ')
+                   .and('contain', 'Email ');
+        currentPage.getContent()
+                   .getCodeInput()
+                   .should('be.visible')
+                   .and('have.value', defaultCode1);
+        currentPage.getContent()
+                   .getEmailInput()
+                   .should('be.visible')
+                   .and('have.value', defaultEmail1);
+
+
+      });
+
+
+
+
+    });
+
+    describe([Tag.SANITY, 'ENG-3299'], 'Click on Buttons', () => {
+
+
+      it([Tag.SANITY, 'ENG-3299'], 'Add a new Sender', () => {
+
+
+        currentPage.getContent().getAddButton().click().wait(1000);
+        currentPage.getContent()
+                   .getCodeInput()
+                   .type(testCode);
+        currentPage.getContent()
+                   .getEmailInput()
+                   .clear()
+                   .type(testEmail);
+        currentPage.getContent().senderSubmit()
+                   .click().wait(500);
+        currentPage.getContent()
+                   .getSenderTable()
+                   .contains(testEmail)
+                   .should('be.visible');
+        cy.validateToast(currentPage);
+
+       cy.senderController()
```

Comment:
Move this call to an _after hook_, as part of the test it might not be executed if there's a failure in the test execution, _after hooks_ are always executed regardless the test exit state